openupgrade_scripts/scripts/website/15.0.1.0/post-migration.py

```diff
@@ -36,6 +44,7 @@ def extract_footer_copyright_company_name(env):
             or f"Copyright © {website.company_id.name}",
         )
         main_copyright_view.with_context(website_id=website.id).arch_db = new_arch
+        _delete_views_recursively(view)
```

Comment:
This is pretty dangerous, you don't have control at this point of the script about what you're going to delete.

Wouldn't the way to go just changing the inheritance to the new views and the deleting the old ones safely?